anaconda.py

```diff
@@ -48,7 +48,13 @@ def exitHandler(rebootData):
     # pylint: disable=used-before-assignment
     if "nokill" in kernel_arguments:
         util.vtActivate(1)
-        print("anaconda halting due to nokill flag.")
+        print("Warning - 'nokill' kernel option detected")
```

Comment:
Let's not introduce warnings. This nokill option is a forgotten leftover, we have been warning users that this is not supported in the past, https://github.com/rhinstaller/anaconda/pull/5723/files#diff-d67591a19442f722c3011c2ead98a9a0f3597f0aecf82267650cbf606aad893b 

And even the warnings have been removed at this point.